phpBB/includes/functions_compatibility.php

```diff
@@ -195,3 +195,69 @@ function get_user_rank($user_rank, $user_posts, &$rank_title, &$rank_img, &$rank
 	$rank_img = $rank_data['img'];
 	$rank_img_src = $rank_data['img_src'];
 }
+
+/**
+ * Wrapper function of \phpbb\request\request::variable which exists for backwards compatability.
+ * See {@link \phpbb\request\request_interface::variable \phpbb\request\request_interface::variable} for
+ * documentation of this function's use.
+ *
+ * @deprecated 3.2.0-a1 (To be removed: 3.4.0)
```

Comment:
Can someone check if this is correct?

I moved this function to `functions_compatibility`, as it is no longer used in phpBB itself.